### DISCUSSION TITLE
how to deploy react to netlify or vercel

### DISCUSSION THREAD

**Discussion by bobwatcherx** [oven-sh/bun#619] [General]
my package.json no scripts start or build

and i install react-scripts with command 
bun add react-scripts but long time ....

is there way to deploy

**Comment by Babailan** [comment]
this question is so complicated  I can't understand but bun can't run script that wasn't in package.json you should provide the npm script and run it on bun. 

like this
![image](https://user-images.githubusercontent.com/83863770/178761652-f9245629-1f6a-4c2a-aebc-a91536d7f2a4.png)
****

**Reply by FrameMuse** [reply]
However there will not be any speed gain (_I didn't notice_) as it still run `node` via `bun`.

**Reply by Babailan** [reply]
@FrameMuse it's complicated but I think it's natural but the performance was greatly improve.
